out/main.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"main.js","sourceRoot":"","sources":["../src/main.ts"],"names":[],"mappings":"AAMA,mBAAmB,OAAc;IAC7B,OAAO,CAAC,GAAG,CAAC,EAAC,KAAK,EAAE,OAAO,CAAC,KAAK,EAAE,SAAS,EAAE,OAAO,CAAC,SAAS,EAAG,IAAI,EAAE,OAAO,CAAC,IAAI,EAAC,CAAC,CAAC;AAC3F,CAAC;AAED,+CAA+C;AAC/C,SAAS,CAAC,EAAC,KAAK,EAAE,MAAM,EAAG,SAAS,EAAE,IAAI,IAAI,CAAC,2BAA2B,CAAC,EAAG,IAAI,EAAE,CAAC,GAAG,EAAG,GAAG,CAAC,EAAE,CAAC,CAAC"}
```

Comment:
This still needs to happen. We don't have an /out/ folder anymore, so delete the /out/ folder and make sure that git notices that this file is gone.